draft-demarco-oauth-status-attestations.md

```diff
@@ -465,6 +465,35 @@ allowing Verifiers to examine and make educated conclusions based on a variety o
 
 TODO Security
 
+# Privacy Considerations
+
+In the design and implementation of Status Attestations, particular attention has been paid to privacy considerations to ensure that the system is respectful of user privacy and compliant with relevant regulations.
+
+## Privacy Consideration: Status Attestation Request Opacity
+
+The request for a status attestation does not carry the digital credential for which the status is being attested, but an opqaue proof of possession of it, evaluable only by the credential issuer that has issued the digital credential where the status attestation is requested for. This design choice is crucial for preventing the possibility of a bogus request being made with the intention of misleading or providing sensitive information to an unintended audience. By decoupling the digital credential from the status attestation request, the system ensures that the request itself does not inadvertently reveal any information about the digital credential or its holder. This approach enhances the privacy and security of the system by ensuring that the attestation process cannot be exploited to gather information about digital credentials or their holders through fabricated requests.
+
+## Privacy Consideration: Opacity of Status Attestation Content
+
+Another key privacy consideration is the design of the status attestation to ensure that it does not contain any information regarding the user or the holder of the digital credential. The status attestation is structured to provide only the necessary information to verify the non-revocation status of a digital credential without disclosing the identity of the holder or any personal information. This is achieved by focusing the attestation content on the credential's status and the verification process, rather than on the credential's holder. This design choice is instrumental in preserving the anonymity of the user, ensuring that the status attestation can be used in a wide range of verification scenarios without compromising the privacy of the individuals involved.
```

Comment:
I'd like to see us move away from "revocation/ suspension" and towards generic attested dynamic state... Lots of use cases for status values that are not booleans.